src/components/Permissions/AppPermissions/AppPermissions.jsx

```diff
@@ -87,6 +88,10 @@ const AppPermissions = ({ t }) => {
 
   const isKonnector = type => type === 'io.cozy.konnectors'
 
+  const filteredRemoteRequests =
+    queryResultRemoteRequests.data?.length > 0 &&
```

Comment:
nit: Here you can move the `length` check in the filter method. This would make the method more secure if you choose to re-use it somewhere else in the future.